ActindoWorkFlow/Operation/02_TrackWorkflowProcess.md

```diff
@@ -0,0 +1,245 @@
+
+[comments]: <> (Check one action or several actions at a time? See Troubleshooting)
+
+
+1. Click the action in the process diagram to be checked.  
+The selected action is displayed in the *Actions* tab.
+
+2. Alternatively, click *Status* drop-down list to filter the processes according to their status.   
+All existing actions with the selected status are displayed.
+
+  The following statuses are displayed:
+
+  - **All**  
+  All existing actions are displayed. This option is displayed per default.
+
+  - **Active**  
+  The actions that are currently running are displayed.
+
+  - **Done**  
+  The actions that are finished are displayed.
+
+  - **Error**  
+  The actions that have produced an error are displayed.
+
+  - **Error; automatic retry**  
+  The actions that have produced an error but are set up for automatic retry are displayed. For detailed information about action settings, see [Link](#to-be-completed).
+
+  - **Suspended**
+
+  - **Process aborted**
+
+  > [Info] Actions can also be filtered with the filter options. Click the ![Filter](/Assets/Icons/Filter.png "[Filter]") (Filter) button to display the filter options.
+
+[comment]: <> (Oli: Statuses Error; automatic retry, Suspended and Process aborted. Wo stellt man "automatic retry" ein?)
+
+3. Check the current action status in the *Status* column.  
+
+  > [Info] Further information about the selected action(s) can be found in the *Logs* tab.
+
+4. Check the number of times an action has been tried in the *Tries* column.
+
+5. Check the number of times an action must be retried after error in the *Maximal retries after error* column.
+
+[comment]: <> (Useful for user to mention those options? Is it possible to configure it somehow?)
+
+6. Check the queue type in the *Queue type* column.  
+The *Default* queue type is normally selected. For detailed information about configuring queue types, see [Configure the queue types](#to-be-competed).
+
+  > [Info] A different queue type can be selected by editing an action, see [Edit an action](01_ManageWorkflows.md#edit-an-action).
+
+7. Check the next point in time when the action is being tried in the *Defer until* column.
+
+[comment]: <> (Useful for user to mention it? Is it possible to configure it somehow?)
+
+
+---
+[comment]: <> (Further info probably only for Troubleshooting)
+
+
+- If an error is displayed, take the necessary action to solve the problem before retrying the action.
+
+  > [Info] Further information about the error displayed can be found in the *Logs* tab.
+
+- Click the checkbox to select the action to be checked.    
+The (editing) toolbar is displayed.
+
+  > [Info] If several action have been filtered according to status, it is also possible to select them all simultaneously. Click the checkbox in the column header to select all actions filtered. For detailed information about this process, see [Retry a process action](#to-be-completed).
+
+- Click the [RETRY] button to execute the action again.
+
+
```

Comment:
möglicher ablauf dieser procedure:

- filter nach status auf all (ich will erstmal alle actions anzeigen)
- actions in diagramm anschauen (status farben erklären)
- action in diagramm auswählen (je nach status wird action in tab angezeigt)
- status und weitere infos zu action in tab actions anschauen (spalten erklären: status, tries, max tries,  queue type, defer until)
- hinweis auf troubleshooting bei error, verweis auf tokens
- ggf. logs einschließen -> mit oli klären (vielleicht auch nur in troubleshooting oder in separatem kapitel)